components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/consent.jsp

```diff
@@ -39,8 +40,15 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title><%=AuthenticationEndpointUtil.i18n(resourceBundle, "wso2.identity.server")%>
-    </title>
+    <!-- title -->
+    <%
+        File titleFile = new File(getServletContext().getRealPath("extensions/title.jsp"));
+        if (titleFile.exists()) {
+    %>
+            <jsp:include page="extensions/title.jsp"/>
+    <%} else {%>
```

Comment:
`<%} else {%>` keep a space in between `<% %>`. E.g. <% } else { %>

Do it on all the pages

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/2426/commits/37e4813fd5f8e9078546c990764edd8d9132b13e